projects/dsp-ui/src/lib/viewer/representation/still-image/still-image.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="still-image-viewer">
+    <!--<div class="navigation left">
```

Comment:
is this (and the corresponding right arrow) here for future use?

Reply:
Yes, I used it in some experiments with sequence mode. We would still have to figure out how to load additional images of a big image collection.

Maybe the IIIF manifests will help. @nemmK is working on them.

Reply:
https://openseadragon.github.io/examples/tilesource-sequence/